src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -830,6 +830,27 @@ public void It_places_package_satellites_correctly(bool crossTarget)
             outputDirectory.Should().HaveFile(@"fr\FluentValidation.resources.dll");
         }
 
+        [WindowsOnlyFact]
+        public void It_places_package_pdb_file_correctly()
+        {
+            var testAsset = _testAssetsManager
```

Comment:
I will add two more unit-tests with 

PublishReferencesDocumentationFiles=true  Xml should exist in the output directory
PublishReferencesDocumentationFiles=false Xml should NOT exist in the output directory